doc/whats-new.rst

```diff
@@ -62,6 +62,9 @@ By `Robin Wilson <https://github.com/robintw>`_.
   overlapping (:issue:`835`) coordinates as long as any present data agrees.
   By `Johnnie Gray <https://github.com/jcmgray>`_.
 
+- Adds DataArray and Dataset methods :py:meth:`cumsum` and :py:meth:`cumprod`.
```

Comment:
Thanks @shoyer, this is fixed now and the link should now work (had to do some minor refractoring).  However, a search for `cumsum` does not return the DataArray and Dataset results for my local test, which is very strange.